src/app/new-page/component/content-multiselect/content-multiselect.component.css

```diff
@@ -0,0 +1,38 @@
+.multiselect {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 15px;
+}
+
+.multiselect-option app-content-new-repeater {
+  margin: 0;
+}
+
+.multiselect[data-columns="1"] app-content-new-multiselect-option {
+  flex: 0 1 100%;
+}
+
+.multiselect[data-columns="2"] app-content-new-multiselect-option {
+  flex: 0 1 calc(50% - 15px);
+}
+
+.multiselect[data-columns="3"] app-content-new-multiselect-option {
+  flex: 0 1 calc(33.3% - 10px);
+}
+
+.multiselect[data-columns="4"] app-content-new-multiselect-option {
+  flex: 0 1 calc(25% - 11.25px);
+
+}
+
+.multiselect[data-columns="5"] app-content-new-multiselect-option {
+  flex: 0 1 calc(20% - 12px);
+}
+
+.multiselect[data-columns="6"] app-content-new-multiselect-option {
+  flex: 0 1 calc(16.7% - 12.5px);
+}
```

Comment:
The number of columns is currently restricted to be 1 to 4 only. So you won't need the 5 & 6 column options